src/seedu/addressbook/data/AddressBook.java

```diff
@@ -99,6 +99,15 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Edits a person in the address book.
+     *
+     * @throws DuplicatePersonException if an equivalent person exists in the address book.
```

Comment:
Hmmm I don't see this exception thrown anywhere. If the exeception is to be thrown, remember to add `throws YourException` at the method signature. =)